python/lsst/cp/verify/verifyFlat.py

```diff
@@ -205,3 +264,46 @@ def verify(self, detectorStatistics, statisticsDictionary):
         success = bool(np.all(list(verifyStats.values())))
 
         return {'EXP': verifyStats}, bool(success)
+
+    def pack(self, statisticsDict, dimensions, outKey):
+        """Repack information into flat tables.
+
+        This method should be redefined in subclasses.
```

Comment:
Should or may?

Reply:
Comment removed here, retained as "should" in the base class (newly added as this method was skipped), with the clarification that that's only if new stats have been calculated that need to be `pack`ed.